<?xml version="1.0"?>
<launch>
<arg name="robot_namespace" default="/"/> 
<arg name="verbosity_level" default="NOTICE"/>
<node name="snapshot_node" pkg="eros" type="snapshot_node"  output="screen" clear_params="true">
        <param name="robot_namespace" value="$(arg robot_namespace)"/>
        <param name="startup_delay"       value="0.0"/>
	<param name="verbosity_level"       value="$(arg verbosity_level)"/>
        <param name="require_pps_to_start"  value="false"/>    
        <param name="loop1_rate"            value="1"/> 
        <param name="loop2_rate"            value="10"/>
        <param name="loop3_rate"            value="5"/>
        <param name="Mode"              value="MASTER"/> <!-- MASTER,SLAVE-->
        <param name="Config_Directory" value="~/config"/> <!-- Where the Snapshot.xml file is located -->
</node>
</launch>
